### DISCUSSION TITLE
Grove RS485 and Pico

### DISCUSSION THREAD

**Discussion by bartvdg** [Seeed-Studio/wiki-documents#412] [Q&A]
I'm new into physical computing and would appreciate all the help here...
I bought the Grove starter kit for Pico but I have challenges to program Pico WH that is connected with Grove RS485 via UART0 (Pin 0 & 1), the RS485 Modbus side is connected to a Modbus device where I want to read register from starting address 0x0100, it has modbus address 1. Can someone help me get started? 
Some additional Modbs device data = Baudrate 19200; Parity : None; Stop bit: 1

Thanks
Bart

**Comment by MatthewJeffson** [comment]
Hi! You can check with our [forum](https://forum.seeedstudio.com/) to see whether there someone using Pico to program. You can also create a topic to see others' help.
But if I may, 
<img width="330" alt="image" src="https://github.com/Seeed-Studio/wiki-documents/assets/104187183/e506fc19-359b-4b8b-9e3d-bd78a48c8f90">
What kinds of collaboration you want to do? Are you interested in being one of our contributors? to make a better wiki platform together:D

**Reply by bartvdg** [reply]
Hi Matthew. I for sure would love to contribute. 
Only a bit frustrated that altough it is a **Starter** kit for **Pico** there is so less doc on how to make the grove accessories work with Pico...
I will give it a try on the forum you refer to, where would bee best place for posting this question?

**Reply by bartvdg** [reply]
created [here](https://forum.seeedstudio.com/t/grove-rs485-with-modbus-device-and-pico-wh/271439)

**Reply by MatthewJeffson** [reply]
Hi, I can see it. And unfortunately the Grove modules which support the"Grove Basic Kit for Raspberry Pi Pico" are all listed in the wiki. If you want to connect with other modules, you may want to try with Dupont thread to connect. 
In the end, I hope someone can help you.
Regards

**Reply by MatthewJeffson** [reply]
Would you mind also take a minute to check with our [contributor program](https://github.com/Seeed-Studio/wiki-documents/issues/336)? We are looking someone who can help us enhance the user experience and provide better support. 
If you are interested, you can also check with the [task list](https://github.com/orgs/Seeed-Studio/projects/6/views/1)
